Library/ViewModels/DashboardViewModel.swift

```diff
@@ -148,6 +154,11 @@ public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewMo
 
     self.project = projectsAndSelected.map(second)
 
+    self.loaderIsAnimating = Signal.merge(
+    //  self.viewDidLoadProperty.signal.map(const(true)),
```

Comment:
Stray comment here.

Reply:
Beyond that, if we don't need it, we can drop the `Signal.merge` ;)